app/pages/intl/test.tsx

```diff
@@ -0,0 +1,17 @@
+import type { NextPage } from 'next'
+import Title from './Title'
+import LanguageSwitcher from './LangugeSwitcher'
+import { ProvideLanguage } from '../../hooks/useLanguage'
+
+const Test: NextPage = () => {
+  return (
+    <ProvideLanguage>
+      <div className={'flex justify-center items-center h-screen'}>
+        <Title name={'Testine Test'} />
+        <LanguageSwitcher/>
+      </div>
+    </ProvideLanguage>
+  )
+}
+
+export default Test
```

Comment:
Should most likely move this to `app/pages/intl.tsx`
As this is only a demo for now it isn't all that important though.

Reply:
_Moved!_